### DISCUSSION TITLE
as a public role, I cannot start an automation?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2046] [Help] [answered]
When I am a public role, I cannot start an automation. It says I don't have the right to do it.

**Comment by mike12345567** [comment, marked as answer]
Hi @oswaldB this is currently something that is not supported, automations require the user triggering it to be logged in. I will discuss this with the team and see if this is something that we want to change/give the ability to change.

**Comment by oswaldB** [comment]
Oki
